lms/static/js/student_profile/views/learner_profile_factory.js

```diff
@@ -72,6 +72,7 @@
                 new FieldsView.DropdownFieldView({
                     model: accountSettingsModel,
                     screenReaderTitle: gettext('Country'),
+                    screenReaderLabel: gettext('Add Country'),
```

Comment:
@symbolist good point.  I actually think it should be the same as the screenReaderTitle, simply "Country".  The fact that you can Add a country to a blank value or Edit an existing one, or even the generic Specify, is implied so there is no need to add any additional context.  A form field with the value "Country" implies that the form field is there to manage the value of that  property.

How is the value ```screenReaderTitle``` currently being used?  Can we just use that var/value?  This applies to the "Preferred Language" field as well.